libraries/GCS_MAVLink/GCS_MAVLink_Parameters.cpp

```diff
@@ -0,0 +1,372 @@
+    // @RebootRequired: True
+    // @User: Advanced
+    AP_GROUPINFO("_PARAMS",   9, GCS_MAVLINK, streamRates[8], DRATE(8)),
+
+    // @Param: _ADSB
+    // @DisplayName: ADSB stream rate
+    // @Description: MAVLink ADSB stream rate
+    // @Units: Hz
+    // @Range: 0 50
+    // @Increment: 1
+    // @RebootRequired: True
+    // @User: Advanced
+    AP_GROUPINFO("_ADSB",   10, GCS_MAVLINK, streamRates[9], DRATE(9)),
+    AP_GROUPEND
+};
+#undef DRATE
+
+static const ap_message STREAM_RAW_SENSORS_msgs[] = {
+    MSG_RAW_IMU,
+    MSG_SCALED_IMU2,
+    MSG_SCALED_IMU3,
+    MSG_SCALED_PRESSURE,
+    MSG_SCALED_PRESSURE2,
+    MSG_SCALED_PRESSURE3,
+#if AP_AIRSPEED_ENABLED
+    MSG_AIRSPEED,
+#endif
+};
+static const ap_message STREAM_EXTENDED_STATUS_msgs[] = {
+    MSG_SYS_STATUS,
+    MSG_POWER_STATUS,
+#if HAL_WITH_MCU_MONITORING
+    MSG_MCU_STATUS,
+#endif
+    MSG_MEMINFO,
+#if !APM_BUILD_TYPE(APM_BUILD_AntennaTracker)
+    MSG_CURRENT_WAYPOINT,
+#endif
+#if AP_GPS_GPS_RAW_INT_SENDING_ENABLED
+    MSG_GPS_RAW,
+#endif
+#if AP_GPS_GPS_RTK_SENDING_ENABLED
+    MSG_GPS_RTK,
+#endif
+#if AP_GPS_GPS2_RAW_SENDING_ENABLED
+    MSG_GPS2_RAW,
+#endif
+#if AP_GPS_GPS2_RTK_SENDING_ENABLED
+    MSG_GPS2_RTK,
+#endif
+    MSG_NAV_CONTROLLER_OUTPUT,
+#if AP_FENCE_ENABLED
+    MSG_FENCE_STATUS,
+#endif
+    MSG_POSITION_TARGET_GLOBAL_INT,
+#if APM_BUILD_TYPE(APM_BUILD_ArduSub)
+    MSG_NAMED_FLOAT
+#endif  // APM_BUILD_TYPE(APM_BUILD_ArduSub)
+};
+static const ap_message STREAM_POSITION_msgs[] = {
+#if AP_AHRS_ENABLED
+    MSG_LOCATION,
+#endif  // AP_AHRS_ENABLED
+    MSG_LOCAL_POSITION
+};
+static const ap_message STREAM_RAW_CONTROLLER_msgs[] = {
+#if APM_BUILD_TYPE(APM_BUILD_Rover)
+    MSG_SERVO_OUT,
+#endif  // #if APM_BUILD_TYPE(APM_BUILD_Rover)
+};
+static const ap_message STREAM_RC_CHANNELS_msgs[] = {
+    MSG_SERVO_OUTPUT_RAW,
+    MSG_RC_CHANNELS,
+#if AP_MAVLINK_MSG_RC_CHANNELS_RAW_ENABLED
+    MSG_RC_CHANNELS_RAW, // only sent on a mavlink1 connection
+#endif
+};
+static const ap_message STREAM_EXTRA1_msgs[] = {
+#if AP_AHRS_ENABLED
+    MSG_ATTITUDE,
+#endif  // AP_AHRS_ENABLED
+#if AP_SIM_ENABLED
+    MSG_SIMSTATE,
+#endif
+#if AP_AHRS_ENABLED
+    MSG_AHRS2,
+#endif  // AP_AHRS_ENABLED
+#if AP_RPM_ENABLED
+    MSG_RPM,
+#endif
+#if APM_BUILD_TYPE(APM_BUILD_ArduPlane)
+    MSG_AOA_SSA,
+#endif
+    MSG_PID_TUNING,
+#if HAL_LANDING_DEEPSTALL_ENABLED && APM_BUILD_TYPE(APM_BUILD_ArduPlane)
+    MSG_LANDING,
+#endif  // HAL_LANDING_DEEPSTALL_ENABLED && APM_BUILD_TYPE(APM_BUILD_ArduPlane)
+#if HAL_WITH_ESC_TELEM
+    MSG_ESC_TELEMETRY,
+#endif
+#if APM_BUILD_TYPE(APM_BUILD_Rover)
+    MSG_WHEEL_DISTANCE,
+#endif  // APM_BUILD_TYPE(APM_BUILD_Rover)
+#if HAL_GENERATOR_ENABLED
+    MSG_GENERATOR_STATUS,
+#endif
+#if AP_WINCH_ENABLED
+    MSG_WINCH_STATUS,
+#endif
+#if HAL_EFI_ENABLED
+    MSG_EFI_STATUS,
+#endif
+#if AP_AIRSPEED_HYGROMETER_ENABLE
+    MSG_HYGROMETER,
+#endif
+};
+static const ap_message STREAM_EXTRA2_msgs[] = {
```

Comment:
a bit of white space between arrays please

Reply:
Done